autogen/dockerversion/dockerversion.go

```diff
@@ -0,0 +1,14 @@
+// Package dockerversion is auto-generated at build-time
```

Comment:
No reason to hide file in "autogen" package. It can just be under "dockerversion" or "version".